extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -45,6 +45,16 @@ message PluginConfig {
   // This is deprecated in favor of AccessLogging enum.
   bool disable_server_access_logging = 1 [deprecated = true];
 
+  // Optional. Allows configuration of the number of log entries request to
```

Comment:
`Allows configuration of the number of log entries request to batch into a single request` This comment is confusing, What it really controls is the size of a LogWrite request.

Reply:
Fixed.